modules/data-streams/src/internalClusterTest/java/org/elasticsearch/datastreams/DataStreamIT.java

```diff
@@ -1857,7 +1857,7 @@ public void testCreateDataStreamAliasWithSameNameAsIndex() throws Exception {
             var e = expectThrows(InvalidAliasNameException.class, indicesAdmin().aliases(aliasesAddRequest));
             assertThat(
                 e.getMessage(),
-                equalTo("Invalid alias name [logs]: an index or data stream exists with the same name as the alias")
+                equalTo("Invalid alias name [logs]: an index, data stream, or ESQL view exists with the same name as the alias")
```

Comment:
Nit: considering that this is a user facing message, do we need to write `ES|QL` instead of `ESQL`?